### DISCUSSION TITLE
5.1 Download - Missing Folders?

### DISCUSSION THREAD

**Discussion by adtrevino4** [now-i-know-my-abcd/docs#64] [Where in the World is…? (Help)] [answered]
Hi all, 

I've been trying to download the ABCD 5.1 data (https://dx.doi.org/10.15154/z563-zd24) by clicking on the _ABCD 5.1 Tabulated Release Data_ link. 

However, when it downloads, there are some folders missing from the "core" ABCD 5.1 folder. Specifically, I'm looking for the mental-health items since some changed since 5.0. 

Is it an issue on my end or is there a different way I should be downloading the data? 

Thank you!

**Comment by laikevin128** [comment, marked as answer]
Maybe you can try using a different browser, or clearing the cache and trying again, the files should be similar.

**Reply by adtrevino4** [reply]
I switched from Safari to Chrome, redownloaded, and now all the folders are there!

Thank you!
